notification-service/src/main/java/com/microcommerce/notificationservice/listener/AccountCreatedEventListener.java

```diff
@@ -14,19 +15,10 @@
 @RequiredArgsConstructor
 public class AccountCreatedEventListener {
 
-    private final EmailSenderService emailSenderService;
+    private final AccountCreatedEventHandlerService accountCreatedEventHandler;
 
-    @KafkaListener(topics = "${kafka.topics.account-topic}", groupId = "group-id-2", containerFactory = "accountCreatedListenerContainerFactory")
+    @KafkaListener(topics = "${kafka.topics.account-topic.name}", groupId = "${kafka.topics.account-topic.group-id}", containerFactory = "accountCreatedListenerContainerFactory")
```

Comment:
Is there any way of message acknowledgment in the situation when the mail is actually delivered. How to handle the situation when the processing of the message from the topic is not successful. If necessary, please make an investigation ticket and propose the best solution.

Reply:
There is a way to manually send acknowledgement to kafka broker using Acknowledgement parameter in listener methods, ticket is created: https://github.com/rbtree/micro-commerce/issues/26